tests/integration_tests/datasources/test_ec2_ipv6.py

```diff
@@ -41,10 +41,14 @@ def test_dual_stack(client: IntegrationInstance):
     assert client.execute("ip6tables -I OUTPUT -d fd00:ec2::254 -j REJECT").ok
     _test_crawl(client, "http://169.254.169.254")
 
-    # Force NoDHCPLeaseError (by removing dhclient) and assert ipv6 still works
+    # Force NoDHCPLeaseError (by removing dhcp clients) and assert ipv6 still
+    # works
     # Destructive test goes last
     # dhclient is at /sbin/dhclient on bionic but /usr/sbin/dhclient elseware
-    assert client.execute("rm $(which dhclient)").ok
+    assert client.execute("rm $(command -v dhclient)").ok
+    # Not preinstalled on all versions
+    client.execute("rm $(command -v dhcpcd)")
```

Comment:
I think we want to only attempt to perform this on Mantic ++ right?
So should we assert we removed this explicitly with the following?

```suggestion
    if CURRENT_RELEASE > MANTIC:
        client.execute("rm $(command -v dhcpcd)").ok
    fi
```

We'd need an import
`from tests.integration_tests.releases import CURRENT_RELEASE, MANTIC`

Reply:
I realize that this review suggestion is basically the same thing as your proposal.  But, it does allow us to assert that we properly removed the dhcp_client as opposed to ignoring the exit code of the rm operation in your approach.

Reply:
done